.gitignore

```diff
@@ -10,3 +10,4 @@ debug
 driver-test-data.tar.gz
 perf
 **mongocryptd.pid
+*.test
```

Comment:
Added because the `go test -c` command produces test binaries ending in `.*test`.